src/lib.rs

```diff
@@ -83,6 +85,17 @@ impl Biscuit {
         ))
     }
 
+    #[wasm_bindgen(js_name = fromBytesWithProvider)]
```

Comment:
we should document that the provider should return a string and not a `PublicKey` like other calls expect (and maybe find a way for other calls to provide either a string or a `PublicKey`

Reply:
the constraint here is that the provider must return a `JsValue`. There might be a way to do that, let me check

Reply:
yup, it works :) [71fad40](https://github.com/eclipse-biscuit/biscuit-wasm/pull/51/commits/71fad400e099dbfbc33e350128f1c60e308fb378)

Reply:
oh, awesome